docs/90-FAQ.md

```diff
@@ -283,6 +291,18 @@ you don't want new clients to be urgent by default put this in your rc.lua:
         end
     end
     client.connect_signal("request::activate", awful.ewmh.activate)
+    
+### How to prevent mouse cursor from jumping to the corner when resizing or moving a client
+
+You can specify the behavior for these layouts:
+
+    awful.layout.suit.floating.resize_jump_to_corner = false
+    awful.layout.suit.tile.resize_jump_to_corner = false
```

Comment:
Isn't this broken right now? I thought I kind of broke that a while back... ( The feature itself still exist, but not that way)

    awful.placement.resize_to_mouse(client.focus)

Kind of does something similar. Else, it would need a `pretend = true` in `awful.mouse.client.resize`. That's not possible right now.